File: PHP PowerPoint Examples/DocumentPropertiesApiDemo/deletePptDocumentProperties.php
<?php

use Spire\Cloud\Powerpoint\Sdk\Api\PropertiesApi;
use Spire\Cloud\Powerpoint\Sdk\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl="https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey,$baseUrl);
$apiInstance = new PropertiesApi($configuration);

$name = "DelProperties.pptx";
$password = null;
$folder = "input";
$storage = null;
$apiInstance->deletePptDocumentProperties($name, $password, $folder, $storage);

File: Python PowerPoint Examples/PropertiesApiDemo/deletePptDocumentProperties.py
import spirecloudpowerpoint
from spirecloudpowerpoint.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey, baseUrl)
api = spirecloudpowerpoint.api.properties_api.PropertiesApi(configuration)

name = "DelProperties.pptx" #Supported formats:ppt/pptx/pptm/pps/ppsx/pot/potx/odp
password = ""
storage = ""
inputFolder = "input"

api.delete_ppt_document_properties(name, password=password, storage=storage, folder=inputFolder)